src/logic/operate.js

```diff
@@ -1,9 +1,14 @@
+/* eslint-disable no-console */
 import Big from 'big.js';
 
 const operate = (numberOne, numberTwo, operation) => {
   let result = Big(0);
   const a = Big(numberOne);
+  console.log(operation);
+  console.log('1', numberOne);
+  console.log('2', numberTwo);
   const b = Big(numberTwo);
+  console.log('b', b);
```

Comment:
- [ ] <b>REQUIRED:</b> Make sure to always keep your code clean by making sure that you delete all debugging codes(console logs) when you no longer need them.